docs/_releases/v2.0.0-pre.4/matchers.md

```diff
@@ -112,6 +112,26 @@ Requires the value to be a `Function`.
 Requires the value to be an `Array`.
 
 
+#### `sinon.match.array.deepEquals(iterable)`
+
+Requires an iterable value to be deep equal another one.
```

Comment:
So no duck typing then? Strictly `Array.isArray()`? I am not super sure what it really brings to the table, since it would leave out `arguments` and other array-like objects, but people can always convert these to proper arrays if need be, so it should be ok. 

And since the test is for an array, I guess it should break on non-arrays.